responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -63,20 +155,21 @@ def __init__(self,
                  classes: Optional[np.ndarray] = None,
                  serializer: Optional[Any] = None,
                  maximum_rows_for_test: int = 5000,
-                 feature_metadata: Optional[FeatureMetadata] = None):
+                 feature_metadata: Optional[FeatureMetadata] = None,
```

Comment:
There are a lot of changes here - I wonder if it makes more sense at this point to have an RAIForecastingInsights, similar to how we have RAITextInsights and RAIVisionInsights classes for text and vision scenarios?  Both the current code and RAIForecastingInsights work, but since forecasting is quite different I really wonder if it makes more sense to just separate it out and have the parameters and serialization logic be more focused on that scenario, especially with all of these kwargs.

Reply:
I had it separated out initially, it's 90% the same if not more. We'd have A LOT of duplication since all the tabular data code is identical. Making it inherit from RAIInsights is also not viable. With some refactoring we could make this much nicer, though.